docs/src/adr/022adr-unification-of-configs-and-options.md

```diff
@@ -0,0 +1,119 @@
+---
+authors: Shon Feder
+last revised: 2022-08-15
+---
+
+# ADR-003: Unify Configuration Management and "Pass Options"
+
+**Table of Contents**
+
+- [Summary (Overview)](#summary)
+- [Context (Problem)](#context)
+- [Options (Alternatives)](#options)
+- [Solution (Decision)](#solution)
+- [Consequences (Retrospection)](#consequences)
+
+## Summary
+
+In the context of extending Apalache's functionality and adding new features\
+facing the need to reason about our program configuration and make execution thread safe\
+we decided for unifying our CLI input with our configuration management system\
+to achieve a more modular architecture and greater static guarantees\
+accepting the development costs and possible increased cost of introducing new CLI inputs in the future.\
```

Comment:
```suggestion
accepting the development costs and possible increased cost of introducing new CLI inputs in the future.
```

Otherwise, this renders as a literal backslash at the end of the paragraph.